src/pages/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import '../App.css';
+import NewListButton from '../components/NewListButton';
+import * as ls from 'local-storage';
+import FirebaseTesting from './FirebaseTest';
+import fb from '../lib/firebase';
+import { Link, Switch, Route, Redirect } from 'react-router-dom';
+import FooterTabs from '../components/FooterTabs';
+import ShoppingList from './ShoppingList';
+import AddItem from './AddItem';
+
+function Home() {
+	const [token, setToken] = useState(ls.get('shoppingListToken'));
```

Comment:
Also, to facilitate moving the switch statement out of this page view, token state would have to be moved up to the App level, and passed down as a prop.